index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="container">
+        <nav class="top-menu">
+            <div class="logo">
+               <a href="#"> Oscar Deus</a>
+            </div>
+            <a class="hamburger">
+                <span class="icon"></span>
+                <span class="icon"></span>
+                <span class="icon"></span>
+            </a>
+        </nav>
+        <section class="home">
+            <h1 class="main-header">
+                I’m Oscar <br> Glad to see you!
+            </h1>
+            <p class="description">
+                I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="contacts">
+                <h2 class="con-header">Let's Connect</h2>
+                <ul class="contact-icons">
+                    <li>
+                        <a href="https://twitter.com/killer_pip">Twiter
+                            <img src="./icons/tw.png" alt="">
```

Comment:
- [ ] I think you mistakenly added a **Twiter** text that altered  your design from the Figma design.

**Your design**
<img width="345" alt="Screenshot 2022-06-30 at 23 26 38" src="https://user-images.githubusercontent.com/31113416/176781472-a7b5fb11-0353-47fc-b603-12d99c8268b9.png">

**Figma design**

![headline-mobile](https://user-images.githubusercontent.com/31113416/176781652-b00a06f2-476f-4fec-8a61-1208f51eb567.png)